src/index.js

```diff
@@ -40,19 +41,67 @@ class Container extends Component {
 }
 
 function bindEvents (options) {
-  events.once('ready', () => options.onReady())
-  events.once('documentCapture', data => options.onDocumentCapture(data))
-  events.once('faceCapture', data => options.onFaceCapture(data))
-  events.once('complete', data => options.onComplete(data))
+  const eventListenersMap = {
+    ready: () => options.onReady(),
+    documentCapture: data => options.onDocumentCapture(data),
+    faceCapture: data => options.onFaceCapture(data),
+    complete: data => options.onComplete(data)
+  }
+
+  _.forIn(eventListenersMap, (listener, event) => {
+    events.once(event, listener)
+  })
+
+  return eventListenersMap;
+}
+
+function unbindEvents (eventListenersMap) {
+  _.forIn(eventListenersMap, (listener, event) => {
+    events.off(event, listener)
+  })
+}
+
+function rebindEvents(newOptions, previousEventListenersMap){
+  if (previousEventListenersMap) unbindEvents(previousEventListenersMap)
+  return bindEvents(newOptions)
+}
+
+/**
+ * Renders the Onfido component
+ *
+ * @param {DOMelement} [merge] preact requires the element which was created from the first render to be passed as 3rd argument for a rerender
+ * @returns {DOMelement} Element which was generated from render
+ */
+const onfidoRender = (options, el, merge) => {
+  return render( <Container options={options}/>, el, merge)
 }
 
+//TODO make reinitialisation work on the same element, the culpirt is Modal
```

Comment:
I always prefer a trackable issue, rather than a TODO comment in the code.

TODOs always get lost 😄